src/Components/helper.js

```diff
@@ -0,0 +1,64 @@
+import current_info from "../info";
+import ordinal from "ordinal";
+
+export const getData = (dail_session=undefined) => {
+	if (dail_session === undefined) {
+		dail_session = current_info.dail
+	}
+	else {
+		document.title = document.title + " - " + ordinal(dail_session) +
+			" Dáil";
+	}
+
+	const info = require('../data/' + dail_session + 'info.json');
+	const members = require('../data/' + dail_session +
+		'members.json');
+	return [info, members]
+};
+
+export const handleCheckedConstituency = (data, constit) => {
+	let result = [];
+	data.forEach(
+		member => {
+			if (member["constituency"].toLowerCase() === constit) {
+				// console.log("constit: ", constit);
+				// console.log("member[\"constituency\"]: ", member["constituency"].toLowerCase());
+				result.push(member)
+			}
+		}
+	);
+	// console.log("result: ", result);
+	return result;
+};
+
+
+
+export const handleCheckedDublin = (data) => {
+	let result = [];
+	data.forEach(
+		member => {
+			// console.log("constit: ", member["constituency"].toLowerCase())
+			const memberConst = member["constituency"].toLowerCase()
+			if (memberConst.startsWith("dublin") || 
```

Comment:
This is the second place this check has to be done. Might be easier to just define an array of the constituencies in Dublin and check against that, easier to maintain.